### ISSUE TITLE
Better compatibility with screen readers like NVDA and JAWS

### ISSUE THREAD

**Issue by supercheetah** [manisandro/gImageReader#426]
I have a blind user, and I was hoping to be able to use this for him to be able to get information out of the PDF and other files he gets, but many of the buttons don't have good descriptions of what they do or any description at all when used within the free NVDA (Non-Visual Desktop Access, [nvaccess.org](https://www.nvaccess.org/about-nvda/)), or within Freedom Scientific's JAWS software (which he primarily uses).  On top of that, the tab order doesn't seem to make sense, at least not in the use-case of a blind user using this with a screen reader anyway.

I have not this with any Linux screen readers, so I'm not sure of the behavior with those, but I would expect them to be the same.

Going with full [508 standards compliance](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-section-508-standards/section-508-standards) would be nice, but just getting this to be descriptive within a screen reader would be enough to fulfill this.

**Comment by manisandro** [comment]
I have little experience with screen readers. Fixing the tab order should be relatively easy. What do you mean by button description? You mean like a text next to the icon in the button? (Which however would break the UI layout in various places), or just the tooltip?

**Comment by supercheetah** [comment]
I think a tool tip will help.  I'm not entirely certain about what the software reads.  I recorded a video (webm) with the audio from NVDA on a fresh instance of gImageReader and tabbed through it with an onscreen keyboard, attached here. A lot of the buttons don't get a good description (just "button"), or lack one entirely (particularly the first one that has focus when gImageReader starts).
[gimagereader4.zip](https://github.com/manisandro/gImageReader/files/3958023/gimagereader4.zip)
